### ISSUE TITLE
Prompt-Travel is not working anymore after I updated the extension

### ISSUE THREAD

**Issue by fatualux** [Kahsolt/stable-diffusion-webui-prompt-travel#35]
Hello, and thanks for your work.

Prompt-Travel works no longer on my configuration after the last update.

`SYSTEM INFORMATION:
Linux
6.4.8-arch1-1 x86_64`

Browser: `Mozilla Firefox 116.0`

Traceback:
```
`2023-08-07 00:09:04,427 - ControlNet - INFO - ControlNet v1.1.234
[ControlNet-Travel] extension Mikubill/sd-webui-controlnet found, ControlNet-Travel loaded :)
Error loading script: controlnet_travel.py
Traceback (most recent call last):
  File "/home/fz/.virtualenv/stablediffusion/modules/scripts.py", line 256, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "/home/fz/.virtualenv/stablediffusion/modules/script_loading.py", line 11, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/fz/.virtualenv/stablediffusion/extensions/stable-diffusion-webui-prompt-travel/scripts/controlnet_travel.py", line 44, in <module>
    from scripts.prompt_travel import *
  File "/home/fz/.virtualenv/stablediffusion/extensions/stable-diffusion-webui-prompt-travel/scripts/prompt_travel.py", line 34, in <module>
    from modules.prompt_parser import DictWithShape
ImportError: cannot import name 'DictWithShape' from 'modules.prompt_parser' (/home/fz/.virtualenv/stablediffusion/modules/prompt_parser.py)

Error loading script: prompt_travel.py
Traceback (most recent call last):
  File "/home/fz/.virtualenv/stablediffusion/modules/scripts.py", line 256, in load_scripts
    script_module = script_loading.load_module(scriptfile.path)
  File "/home/fz/.virtualenv/stablediffusion/modules/script_loading.py", line 11, in load_module
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/fz/.virtualenv/stablediffusion/extensions/stable-diffusion-webui-prompt-travel/scripts/prompt_travel.py", line 34, in <module>
    from modules.prompt_parser import DictWithShape
ImportError: cannot import name 'DictWithShape' from 'modules.prompt_parser' (/home/fz/.virtualenv/stablediffusion/modules/prompt_parser.py)`
```
Could you please give me any support?
Thank you in advance

**Comment by Kahsolt** [comment]
Please check your `sd-webui` version, and notice the "Change Log - Compatibility" section of `README.md`

This repo tracks changes from other repos to stay compatible with.
If you update this repo, you should also update at leaset the `sd-webui` to the version that documented.

**Comment by fatualux** [comment]
you are right, I was running an outdated version of sd--webui.
Thank you.
I did not realize it because, being it in a virtualenv, it is not managed by my system's package manager.
The strange thing is that if I gave `git pull` inside the directory I got `Already up to date.`
I had to remove all of the directories and files, replace them with the cloned new ones, then `python -m pip install -r requirements.txt` and `python  launch.py`.
Anyway, thank you so much, you solved the pronlem.
And thanks for your awesome work.
